### ISSUE TITLE
If I start to write a note and select the text, to deselect the text, only works clicking on the text, but should be possible to deselect clicking in any part of the window 

### ISSUE THREAD

**Issue by bitcoinuser** [PrimalHQ/primal-android-app#110]
Hi,

If I start to write a note and select the text, to deselect the text, only works clicking on the text, but it should be possible to deselect clicking in any part of the window. 

![photo_5145688047348722847_y](https://github.com/PrimalHQ/primal-android-app/assets/1178856/bc4a6463-16a3-48cf-85c3-cd148b6bf5f4)

**Comment by AleksandarIlic** [comment]
Yeah, I agree. Unfortunately this is what we get from the system at the moment. Hopefully they'll fix it at some point.
